File: CS50x solutions/week_6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

height_list = [1, 2, 3, 4, 5, 6, 7, 8]

while 1:
    height = get_int("height: ")
    if height in height_list:
        break
for row in range(height):
    for collum in range(height):
        if row + collum < height - 1:
            print(" ", end = '')
        else:
            print("#", end = '')
    print()

File: week_1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    // for each row
    for (int row = 0; row < height; row++)
    {
        // for each collum
        for (int collum = 0; collum < height; collum++)
        {
            if (row + collum < height - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        // for going to the next ligne after finishing row 0
        printf("\n");
    }
}